### DISCUSSION TITLE
Notification of partial available - ep1 say of series request.

### DISCUSSION THREAD

**Discussion by mrjohnpoz** [sct/overseerr#1024] [Feature Requests]
Would it be possible, I didn't see where you could set this.  So user request series xyz, and only ep1 is out for example.  Can they get a notification when ep1 is available, then when ep2, etc.  Vs not getting notification until the whole series is available.

**Comment by sct** [comment]
This is planned.

**Comment by st-sha** [comment]
@sct is this still planned? I would love to be able to have notifications sent to users when all available episodes are available rather than needing to wait until the entire season has aired.
